packages/perspective-jupyterlab/package.json

```diff
@@ -39,16 +35,16 @@
         "@finos/perspective-viewer": "^1.0.1",
         "@finos/perspective-viewer-d3fc": "^1.0.1",
         "@finos/perspective-viewer-datagrid": "^1.0.1",
-        "@jupyter-widgets/base": "^3.0.0 || ^4.0.0",
+        "@jupyter-widgets/base": "^4.0.0",
```

Comment:
This is a breaking change just to note

Reply:
I need to check - this branch has been under development for several weeks and I don't recall why I removed this, so I'll need to revert and verify.  I suspect it was because `^3.0.0` does not work even in already-released versions - if so, this is only a breaking change by technicality.

Reply:
I remember now - this dep resolved to only `"^4.0.0"` in our `yarn.lock`, hence we actually have no idea if it works for `"^3.0.0"`.  Setting this to `"^3.0.0"` does cause yarn install issues, so I'll likely still remove this, but there's no reason to intermix this change with the other 10k lines of esbuild changes, so I've backed this out for now.